packages/utils/src/date-utils/getTimeFromPrevious.ts

```diff
@@ -1,45 +1,34 @@
-import { checkIsNextDay } from './checkIsNextDay.js';
+import type { OntimeEvent } from 'ontime-types';
+
 import { dayInMs } from './conversionUtils.js';
 
 /**
  * Utility returns the time elapsed (gap or overlap) from the previous
  * It uses deconstructed parameters to simplify implementation in UI
  */
 export function getTimeFromPrevious(
```

Comment:
question: do we still need this function?

Reply:
used in `rundownCache` and `timeline.utils.ts`
might be able to remove it from timeline at some point...